app/services/s3_query_service.rb

```diff
@@ -19,31 +19,44 @@ def self.post_curation_config
     configuration.post_curation
   end
 
+  def self.preservation_config
+    configuration.preservation
+  end
+
   attr_reader :part_size, :last_response
 
   ##
   # @param [Work] model
-  # @param [Boolean] pre_curation
+  # @param [Boolean] pre_curation True to use pre_curation S3 buckets, false to use post_curation S3 buckets.
+  # @param [Boolean] preservation True to use our preservation buckets in AWS S3
+  #                               (when true it takes precedence over the pre_curation parameter)
   # @example S3QueryService.new(Work.find(1), true)
-  def initialize(model, pre_curation = true)
+  def initialize(model, pre_curation = true, preservation = nil)
```

Comment:
I wonder instead of two flags if it would make more sense to pass a mode parameter that could have the value or precuration, postcuration, or preservation.  It just seems weird for precuration to be true, but still be false if preservation is true.

```suggestion
  def initialize(model, mode = "pre_curation")
```